.github/workflows/integration-tests.yml

```diff
@@ -35,6 +35,7 @@ jobs:
             SATURN_NETWORK=local
             ORCHESTRATOR_URL=http://localhost
             LOG_INGESTOR_URL=http://localhost
+            LASSIE_EXCLUDE_PROVIDERS=bafzbeigai3eoy2ccc7ybwjfz5r3rdxqrinwi4rwytly24tdbh6yk7zslrm
```

Comment:
Why here?

Reply:
I think this was related to nginx hanging 🤔

Reply:
Something we did with @guanzo when Lassie was failing so we can pass a random CID but prob not needed after your fix

Reply:
yeah this isn't needed anymore, the Dockerfile passes a default provider so lassie doesn't crash on startup